config.example/group_vars/all.yml

```diff
@@ -3,13 +3,6 @@
 
 # Account to use for SSH connections
 ansible_user: ubuntu
```

Comment:
`ansible_user` needs to go away. For SSH connections, you can configure your SSH client, or pass `-u <user>` to `ansible-playbook`